doc/manual/noteworthy-differences.rst

```diff
@@ -251,3 +251,75 @@ Noteworthy differences from Python
+  There are no numeric literal suffixes, such as L, LL, U, UL, ULL to indicate unsigned
+  and/or signed vs. unsigned.
+  Decimal literals are always signed, and hexadecimal literals (which start with 0x like C/C++),
+  are unsigned (except if they are >128 bits, in which case they become signed BigInt type).
+  Hexadecimal literals also, unlike C/C++/Java and unlike decimal literals in Julia,
+  have a type based on the *length* of the literal, including leading 0s.  For example,
+  ``0x0`` and ``0x00`` have type UInt8, ``0x000`` and ``0x0000`` have type UInt16, then
+  literals with 5 to 8 hex digits have type UInt32, 9 to 16 hex digits type UInt64, and more than
+  16 hex digits end up a signed BigInt type.  This needs to be taken into account when defining
+  hexadecimal masks, for example `~0xf` is very different from `~0x000f`.
+  Unlike either decimal integer or hexadecimal literals, floating point literals are always
+  64-bits, Float64, and don't get promoted to the BigFloat type.  This is closer in
+  behavior to C/C++.
+- String literals can be delimited with either `"`  or `"""`, `"""` delimited literals can contain `"` characters without
+  quoting it like `"\\\""`
+  They can also have string interpolation, indicated by $variablename or $(expression),
+  which evaluates the variable name or the expression in the context of the function.
+- `//` indicates a Rational number, and not a single-line comment (which is # in Julia)
+- `#=` indicates the start of a multiline comment, and `=#` ends it.
+- Functions in Julia return values from their last expression(s) or the ``return``
+  keyword.  Multiple values can be returned from functions and assigned as tuples, e.g.
+  ``(a, b) = myfunction()`` or ``a, b = myfunction()``, instead of having to pass pointers
+  to values as one would have to do in C/C++ (i.e. ``a = myfunction(&b)``.
+- Julia discourages the used of semicolons to end statements. The results of
+  statements are not automatically printed (except at the interactive prompt),
+  and lines of code do not need to end with semicolons. :func:`println` or
+  :func:`@printf` can be used to print specific output.
+- In Julia, the operator :obj:`$` performs the bitwise XOR operation, i.e. :obj:`^`
+  in C/C++.  Also, the bitwise operators do not have the same precedence as C/++.
+  They can operate on scalars or element-wise across arrays and can be used to
+  combine logical arrays, but note the difference in order of operations:
+  parentheses may be required (e.g., to select elements of ``A`` equal to 1 or
+  2 use ``(A .== 1) | (A .== 2)``).
+- Julia's :obj:`^` is exponentiation, not bitwise XOR as in C/C++ (use :obj:`$` in Julia)
+- Julia's ``->`` creates an anonymous function, it does not access a member via a pointer.
+- Julia does not require parentheses when writing ``if`` statements or
+  ``for``/``while`` loops: use ``for i in [1, 2, 3]`` instead of
+  ``for (i in c(1, 2, 3))`` and ``if i == 1`` instead of ``if (i == 1)``.
```

Comment:
this first example is R syntax, right? probably not appropriate for this section